streetwise/api/images.py

```diff
@@ -46,12 +56,16 @@ class ImageRandom(Resource):
     @ns.doc('random_images')
     @ns.marshal_list_with(ImageModel)
     def get(self, campaign_id):
-        if campaign_id is None or campaign_id == 'null':
-            print("Null campaign requested: selecting default (1)")
-            campagn_id = 1
+        # Prime the image counter if needed
+        if not 'image_counter' in g:
+            g.image_counter = init_image_counter()
```

Comment:
This is rather "buried", and I would have preferred to have it happen in the init of the app, but I couldn't get it working in `__init__.py` - so here it is. Some unlucky user may get a timeout if they open the un-primed app and the init takes a long time.